vortex-array/src/compute/list.rs

```diff
@@ -77,8 +77,12 @@ pub fn list_contains(array: &dyn Array, value: Scalar) -> VortexResult<ArrayRef>
     // Fast path: no elements match.
     if let Some(pred) = matches.as_constant() {
         if matches!(pred.as_bool().value(), None | Some(false)) {
-            // TODO(aduffy): how do we handle null?
```

Comment:
@a10y did you have some concerns here with this approach?

Reply:
No i think you're right, this looks like it was a bug.

Reply:
Actually, I'm not sure this quite fixes it? The constant array is still a constantarray of `false?` but it doesn't carry any validity buffer. Do we need to build a sparsearray with null fill value?

Reply:
Haha that's another bug. I was fixing type, didn't realize this handles None. How about a constant null array?